ios/accessibility/accessibility_control.go

```diff
@@ -19,10 +17,10 @@ func (a ControlInterface) readhostAppStateChanged() {
 		msg := a.channel.ReceiveMethodCall("hostAppStateChanged:")
 		stateChange, err := nskeyedarchiver.Unarchive(msg.Auxiliary.GetArguments()[0].([]byte))
 		if err != nil {
-			panic(err)
+			logrus.WithError(err).Error("Failed unarchiving NSKeyedArchiver plists")
```

Comment:
Can you please change back `logrus` to log? 
this is a little verbose for me :-)

Reply:
sure! done it in all the other places as well, let me know if it looks good or I missed smt :D